src/main/java/ch/njol/skript/effects/EffPotion.java

```diff
@@ -43,14 +44,16 @@
 @Examples({"apply swiftness 2 to the player",
 		"remove haste from the victim",
 		"on join:",
-		"	apply potion of strength of tier {strength.%player%} to the player for 999 days"})
-@Since("2.0, 2.2-dev27 (ambient and particle-less potion effects), 2.5 (replacing existing effect)")
+		"	apply potion of strength of tier {strength.%player%} to the player for 999 days",
```

Comment:
\t here instead of an actual tab? since you're already editing this file ;D

Reply:
Good call ... done!